scene/resources/curve.cpp

```diff
@@ -1461,7 +1464,7 @@ void Curve3D::_bake() const {
 			if (d > bake_interval) {
 				// OK! between P and NP there _has_ to be Something, let's go searching!
 
-				int iterations = 10; //lots of detail!
+				const int iterations = 10; //lots of detail!
```

Comment:
```suggestion
				const int iterations = 10; // Lots of detail!
```
Our code style is sentence styled casing.

Reply:
done.